Example/Example/ComicsViewController.swift

```diff
@@ -19,6 +19,12 @@ class ComicsViewController: ExampleViewController, BothamCollectionViewControlle
     override func viewDidLoad() {
         collectionView.accessibilityLabel = "Comics CollectionView"
         collectionView.delegate = self
+        dataSource.reuseIdentifierForItem = { item in
+            if case .Featured = item {
+                return "FeaturedComicCollectionViewCellReuseIdentifier"
+            }
+            return "ComicCollectionViewCellReuseIdentifier"
+        }
```

Comment:
Is this why we want to implement something like: https://github.com/pedrovgs/Renderers? Or is this approach enough?